test/functions_test.rb

```diff
@@ -151,10 +162,14 @@ def nice_color_argument(color)
         return Script::String.new(color.to_s, :string)
       end
 
-      def returns_a_color()
+      def returns_a_color
         return Script::Color.new(red: 0, green: 0, blue: 0)
```

Comment:
Just curious: how come we chose to have a slightly different module namespace for values?

i.e. why not `Script::Value::Color`?